site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -199,3 +200,14 @@
         });
     });
 </script>
+<script type="text/javascript">
+    function submitNearestForm(e, textarea){
+        if ((e.keyCode == 10 || e.keyCode == 13) && e.ctrlKey) {//ctrl+enter keys pressed
```

Comment:
I have a feeling this isn't going to work on Mac, e.ctrlkey is windows hardware-specific you're probably going to need to do `e.metaKey || e.ctrlKey`